src/main/java/dan200/computercraft/core/apis/OSAPI.java

```diff
@@ -199,11 +205,14 @@ public final void cancelTimer( int token )
 
     /**
      * Sets an alarm that will fire at the specified world time. When it fires,
-     * an alarm event will be added to the event queue.
+     * an {@code alarm} event will be added to the event queue with the ID
+     * returned from this function as the first parameter.
      *
      * @param time The time at which to fire the alarm, in the range [0.0, 24.0).
-     * @return The ID of the alarm that was set.
+     * @return The ID of the new alarm. This can be used to filter the
+     *   {@code alarm} event, or @{os.cancelAlarm|cancel the alarm}.
```

Comment:
```suggestion
     *   {@code alarm} event, or {@link #cancelAlarm cancel the alarm}.
```

Haven't tested, but _should_ work.